ml-agents-envs/mlagents_envs/side_channel/environment_parameters_channel.py

```diff
@@ -35,3 +42,58 @@ def set_float_parameter(self, key: str, value: float) -> None:
         msg.write_int32(self.EnvironmentDataTypes.FLOAT)
         msg.write_float32(value)
         super().queue_message_to_send(msg)
+
+    def set_uniform_sampler_parameters(
+        self, key: str, min_value: float, max_value: float, seed: int
```

Comment:
Should the seed be an optional argument ?

Reply:
If a user doesn't specify the seed, I just send whatever was chosen by python as a run seed. This way, I don't need to sample a seed on the C# side if none is specified.